recipes/MEAutility/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "MEAutility" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7cd1f723992aa7366f1cbbf8f2843e1f61d8123b459435653860a3b544bc752f"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - matplotlib
+    - numpy
+    - pip
+    - python
+    - pyyaml
+  run:
+    - matplotlib
+    - numpy
+    - python
+    - pyyaml
+
+test:
+  requires:
+    - pytest
+  imports:
+    - MEAutility
+    - MEAutility.tests
+  commands:
+    - py.test -vsx --pyargs MEAutility
+
+about:
+  home: "https://github.com/alejoe91/MEAutility"
+  license: "GNU General Public v3 (GPLv3)"
```

Comment:
please use a license identifier from the following list:

https://spdx.org/licenses/